specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-09-01-preview/examples/QueryPackQueriesPut.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "api-version": "2019-09-01-preview",
+    "subscriptionId": "86dc51d3-92ed-4d7e-947a-775ea79b4918",
+    "resourceGroupName": "my-resource-group",
+    "queryPackName": "my-querypack",
+    "queryId": "a449f8af-8e64-4b3a-9b16-5a7165ff98c4",
+    "QueryPayload": {
+      "name": "a449f8af-8e64-4b3a-9b16-5a7165ff98c4",
+      "type": "microsoft.insights/querypacks/queries",
```

Comment:
I think your fix for the circular reference messed things up.  "name" and "type" should not be included here.  Name comes from queryPackName and type is readonly

Reply:
fixed.

Reply:
fixed